src/components/styles/filters.tsx

```diff
@@ -4,7 +4,13 @@ import { Input } from "antd";
 export const FiltersWrapper = styled.div`
   display: flex;
   margin-bottom: 12px;
+  align-items: center;
 `;
+
+export const FiltersWrapperSpaceBetween = styled(FiltersWrapper)`
+  justify-content: space-between;
+`;
```

Comment:
same comment here about having styled components in the shared folder that are slight modifications of other shared components

Reply:
I'll remove this one from the shared components folder. It may not be used again for a little while.